packages/salesforcedx-vscode-lwc/src/messages/i18n.ts

```diff
@@ -20,7 +20,7 @@ export const messages = {
   command_canceled: '%s was canceled.',
   force_lightning_lwc_start_text: 'SFDX: Start Local Development Server',
   force_lightning_lwc_start_not_found:
-    'To run this command, install the @salesforce/lwc-dev-server plugin. For more info, see [https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.get_started_local_dev](https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.get_started_local_dev).',
+    'To run this command, install the @salesforce/lwc-dev-server plugin. For more info, see [https://developer.salesforce.com/tools/vscode/en/localdev/set-up-lwc-local-dev](https://developer.salesforce.com/tools/vscode/en/localdev/set-up-lwc-local-dev).',
```

Comment:
I'm sure this is my fault - but why don't we go with: For more info, see [Set Up LWC Local Development](url) ??

Reply:
Done. Thanks!